### ISSUE TITLE
Consider option for colorblind users

### ISSUE THREAD

**Issue by lmsurpre** [bstnfrmry/hanabi#167]
Played this game with physically-separated family tonight and thoroughly enjoyed.
One minor issue we hit is that one player was red/green colorblind and had a hard time distinguishing these hues.

Any chance of introducing a more color-blind friendly palette, perhaps as a setting for a room?

Alternatively, I think the physical version of this game has shapes as well as colors...any plans to introduce something similar?

**Comment by sarahmamy** [comment]
Hello @lmsurpre! Thanks a lot for proposing this issue. We have been discussing with @benjamintd and @bstnfrmry  about a colorblind option and the first solutions were conflicting with the reduced space on Mobile.

I'm seduced by your proposal to have it as a setting. Maybe similarly to switching language, one user could switch into a colorblind mode.

We'll look into it. That's something we don't want to lay behind.

**Comment by bstnfrmry** [comment]
Hey @lmsurpre!

I added a color-blind option when creating a new game. If you get the chance, could you make a test run and let me know if it's better?

It's not live yet but you can test it here: https://hanabi-git-color-blind-mode.hanabi.now.sh/new-game.

Here's a preview:

![image](https://user-images.githubusercontent.com/1478542/83460530-e8593000-a466-11ea-8417-a5e6de3d6577.png)

And with a protanopia filter:

![image](https://user-images.githubusercontent.com/1478542/83460554-f6a74c00-a466-11ea-8fa4-022de3a03b44.png)

**Comment by lmsurprenant** [comment]
Awesome!  I’m not sure if we’re quite ready for “advanced” but I think we’re gonna give it a go tomorrow night; thanks for the impressively quick responses!

**Comment by lmsurpre** [comment]
We got a chance to play this last friday and our colorblind friend said it was much easier for him, thanks again!

We did notice a few peculiarities that I'll highlight here, but even without fixing these I think it was more playable for him (and still playable for us as well).

1. circle instead of shape when you've been hinted a color (this screenshot from firefox)
![hanabi](https://user-images.githubusercontent.com/5478798/84334578-b0d23e00-ab60-11ea-9a1b-585852f3597b.png)

2. oversized shapes/strange coloring on mobile safari
![hanabi2](https://user-images.githubusercontent.com/5478798/84334737-1c1c1000-ab61-11ea-8d4c-d22e8f8210ba.jpg)

3. minor stacking issue in the piles (visible in both screenshots)

Thanks again!
